codecov.yml

```diff
@@ -0,0 +1,12 @@
+codecov:
+  token: 60a8d8f2-fa0c-4bd5-8631-539c4e6bcf37
+
+# see https://wiki.audaxhealth.com/display/ENG/Codecov for justification of customization
```

Comment:
This won't be visible to third party users of this library.

Reply:
Oops... I'll remove that